docs/DeveloperGuide.md

```diff
@@ -732,19 +736,45 @@ testers are expected to do more *exploratory* testing.
-       
-    3. Test case: e.g. `find t/activity ti/Meeting`
-        Expected: All activities whose title or description contains the substring "Meeting" (case-insensitive) will be displayed. 
-       
-    4. Other incorrect find commands to try: `find t\patient Joe` 
-        Expected: Error message as a flag is not specified prior to the keyword. 
+### Editing an activity
+
+1. Editing an existing activity
+
+    1. Prerequisites: Clear the entire activity using `clear t/activity` command. 
+   Add a new activity using `add t/activity` command to ensure at least 1 activity with id `A001` is there. Check that it exists using `list t/activity`. Please do the test sequentially. 
+
+    2. Test case: `edit t/activity i/A001 ti/Another new title`<br>
+       Expected: Activity whose id `A001` has its title changed to "Another new title"
+
+    3. Test case: `edit t/activity i/A001 s/17/10/2021 14:00 e/17/10/2021 15:00`<br>
+       Expected: Activity whose id `A001` has its start time changed to "17-10-2021 14:00" and end time to "17-10-2021 15:00"
+
+    4. Test case: `edit t/activity i/A001 s/17/10/2021 18:00 e/17/10/2021 15:00`<br>
+       Expected: Error message shows start time must be before end time. 
+
+    5. Test case: Add another activity using `add t/activity s/15/09/2022 14:00 e/15/09/2022 15:00 ti/Meeting with Mr. Y` and then run 
+   `edit t/activity i/A001 s/15/09/2022 14:00 e/15/09/2022 15:00`<br>
+       Expected: Error message shows the activity's timing is conflicting with another activity.
+
+    6. Other incorrect delete activity commands to try: `edit t/activity i/a001 pi/p001` (cannot change patient id), `delete t/activities`, `edit t/activity` (no parameters), `...` <br>
+      Expected: Error message shown in the feedback box.
+
+### Creating A Referral
+
+1. Creating a referral using the template available. 
+
+    1. Prerequisites: Check that you have `[JAR Location]/data` folder, it should be created after you run **GoMedic** for the first time. 
+   Clear the entire patient and activity using `clear t/patient` and `clear t/doctor` respectively. Run the following commands to add 1 patient and doctor using
+   `add t/patient n/John Doe p/98765432 a/45 b/AB+ g/M h/175 w/70 m/heart failure m/diabetes` and `add t/doctor n/John Smith p/98765432 de/Cardiology` respectively. Check that patient whose id `P001` and doctor whose id `D001` exists
+   using `list t/patient` and `list t/doctor` respectively. Also use this default profile by inputting this command `profile n/John Smith p/Senior Resident de/Cardiology o/NUH`.
+   
+    2. Test case: `referral ti/Referral di/D001 pi/P001 d/It looks like there may be a small tear in his aorta.`<br>
+       Expected: A new referral called `Referral.pdf` is created in the `data` folder. The file should look like the following image but the date should be the date where you run the referral command.
+    
+    ![referral](images/referral.png)
```

Comment:
I think the details in this photo doesn't match the patient, doctor, and referral description in the prerequisite and test case.

Reply:
oh oops i might have overlooked it, lemme fix it